### PR TITLE
WSI-1848: Fix package vesion.

### PR DESCRIPTION
**Pull request by SvyatoslavZlatous**

### CODE REVIEW

File: force-app/main/default/classes/plugin/EmailLoggerPlugin.cls [hunk 1]
```diff
@@ -37,47 +40,45 @@ public without sharing class EmailLoggerPlugin implements LoggerPlugin.Triggerab
...
-                }               
-            }
-            if (logsSent) { 
-                flagLogsSent(fullLogs);
-                isFirstExecution = false;
-                update fullLogs;   
+                    Messaging.SendEmailResult [] emailResults = Messaging.sendEmail(emailLogs);
+                    processEmailResults(emailResults);
+                } catch (Exception ex) {
+                    // prevent logger framework from not saving logs
```
Comment (mykhaylo-partyka) [inline force-app/main/default/classes/plugin/EmailLoggerPlugin.cls:71]: Lets store the error via System.debug() at least

File: force-app/main/default/classes/plugin/EmailLoggerPlugin.cls [hunk 2]
```diff
@@ -187,24 +191,46 @@ public without sharing class EmailLoggerPlugin implements LoggerPlugin.Triggerab
...
+        if (logsSent) {
+            setLogsSentSucess(logs);
+        } else {
+            setLogsSentFailure(logs);
+        }
+        isFirstExecution = false;
+        update logs;
+    }
+
+    private static void setLogsSentSucess(List<Log__c> logs) {
```
Comment (mykhaylo-partyka) [inline force-app/main/default/classes/plugin/EmailLoggerPlugin.cls]: The method name is not clear until you look into its code. Let's rename it to something like markLogsAsSent() and markLogsAsFailedToSend() or 
setSendStatusSucceeded() and setSendSatatusFailed()

File: force-app/main/default/classes/plugin/EmailLoggerPlugin.cls [hunk 3]
```diff
@@ -37,47 +40,45 @@ public without sharing class EmailLoggerPlugin implements LoggerPlugin.Triggerab
...
-            List<Log__c> fullLogs = requeryLogs(logIds);
-            List<Messaging.SingleEmailMessage> emails = new List<Messaging.SingleEmailMessage>();
-            if (!EMAIL_RECIPIENTS.isEmpty()) {
-                String orgWideEmailName = LoggerParameter.getString('EmailLoggingOrgWideAddress', '');
-                Id orgWideEmailId = EmailLoggerUtils.getOrgWideEmailAddress(ORG_WIDE_EMAIL_NAME)?.Id;
+            List<Log__c> fullLogs = requeryFullLogs(logIds);
+            List<Messaging.SingleEmailMessage> emailLogs = new List<Messaging.SingleEmailMessage>();
+            if (!LOG_RECIPIENTS.isEmpty()) {
+                String defaultSenderName = LoggerParameter.getString('EmailLoggingOrgWideEmailAddressName', null);
+                Id defaultSenderId = EmailLoggerUtils.getOrgWideEmailAddress(defaultSenderName)?.Id;
```
Comment (mykhaylo-partyka) [inline force-app/main/default/classes/plugin/EmailLoggerPlugin.cls:50]: if defaultSenderId == null. Will it work or fail?

Comment (SvyatoslavZlatous) [reply force-app/main/default/classes/plugin/EmailLoggerPlugin.cls:50]: It will work.

**Pull request merged**